cs-ssh/src/test/java/io/cloudslang/content/ssh/services/impl/SSHServiceImplTest.java

```diff
@@ -1,10 +1,7 @@
 package io.cloudslang.content.ssh.services.impl;
 
 import com.hp.oo.sdk.content.plugin.GlobalSessionObject;
-import com.jcraft.jsch.ChannelShell;
-import com.jcraft.jsch.JSch;
-import com.jcraft.jsch.JSchException;
-import com.jcraft.jsch.Session;
+import com.jcraft.jsch.*;
```

Comment:
Prefer qualified imports instead of star imports as it is more clear what classes are actually used